* This patch is only mandatory for LCD4linux v0.10.0, as it's been fixed in the CVS

--- lcd4linux-0.10.0/lcd4linux_i2c.h.orig	Wed Jun  1 14:09:13 2005
+++ lcd4linux-0.10.0/lcd4linux_i2c.h	Wed Mar 29 14:05:56 2006
@@ -78,7 +78,7 @@
 
 /* Transfer num messages.
  */
-extern int i2c_transfer(struct i2c_adapter *adap, struct i2c_msg msg[], int num);
+extern int i2c_transfer(struct i2c_adapter *adap, struct i2c_msg *msg, int num);
 
 /*
  * Some adapter types (i.e. PCF 8584 based ones) may support slave behaviuor. 
